spec/yaml/swiftnav/sbp/observation.yaml

```diff
@@ -1755,3 +1755,33 @@ definitions:
           type: double
           units: rad
           desc: Argument of perigee at instant of t_lambda
+          
+ - MSG_GLO_BIASES:
+    id: 0x0075
```

Comment:
This looks good to me, catches everything we need from the 1230 and I think it's right to leave out the alignment field. If they're unaligned we'll just ignore them.